lib/lambda_ethereum_consensus/state_transition/operations.ex

```diff
@@ -1039,4 +1039,43 @@ defmodule LambdaEthereumConsensus.StateTransition.Operations do
 
     updated_validators
   end
+
+  @spec process_operations(BeaconState.t(), BeaconBlockBody.t()) ::
+          {:ok, BeaconState.t()} | {:error, binary}
+  def process_operations(state, body) do
+    # Ensure that outstanding deposits are processed up to the maximum number of deposits
+    with {:ok} <- verify_deposits(state, body) do
```

Comment:
The dialyzer error is because of this:

```suggestion
    with :ok <- verify_deposits(state, body) do
```

Reply:
yeah, I changed the return type to `:ok` from `{:ok}`. But still the build is failing 😭

Reply:
oops 🫠 . Fixed it señor.